platform/darwin/src/MGLGeoJSONSource.h

```diff
@@ -15,16 +15,16 @@ extern NSString * const MGLGeoJSONClusterOption;
 
 /**
  An `NSNumber` object containing an integer; specifies the radius of each
- cluster when clustering points, measured in <sup>1</sup>/<sub>512</sub>ths of a
- tile. The default value is 50.
+ cluster if clustering is enabled. A value of 512 indicates a radius equal to 
+ the width of a tile. The default value is 50.
  */
 extern NSString * const MGLGeoJSONClusterRadiusOption;
 
 /**
  An `NSNumber` object containing an integer; specifies the maximum zoom level at
- which to cluster points. Defaults to one zoom level less than the value of
- `MGLGeoJSONMaximumZoomLevelOption`, so that at the last zoom level, the
- features are not clustered.
+ which to cluster points if clustering is enabled. Defaults to one zoom level 
+ less than the value of `MGLGeoJSONMaximumZoomLevelOption` so that, at the last
```

Comment:
While you’re here: s/last zoom level/maximum zoom level/ to avoid confusion with “previous zoom level”.